src/Symfony/Bundle/WebProfilerBundle/DependencyInjection/Configuration.php

```diff
@@ -33,7 +33,16 @@ public function getConfigTreeBuilder(): TreeBuilder
 
         $treeBuilder->getRootNode()
             ->children()
-                ->booleanNode('toolbar')->defaultFalse()->end()
+                ->arrayNode('toolbar')
+                    ->info('Profiler toolbar configuration')
+                    ->canBeEnabled()
+                    ->children()
+                        ->booleanNode('ajax_replace')
```

Comment:
the XSD schema needs to be updated to support this new config for people using XML

Reply:
unfortunately i don't really know what i'm doing. searched for references in the code base and tried an online tool.

is it possible that this would be a breaking change for xml config since it was an attribute before and now is an element? :thinking:

Reply:
we need to keep the attribute in the XSD for BC reasons

Reply:
done!